UndertaleModLib/Util/QoiConverter.cs

```diff
@@ -66,12 +66,13 @@ public static SKBitmap GetImageFromStream(Stream s)
         /// <summary>
         /// Creates a <see cref="SKBitmap"/> from a <see cref="ReadOnlySpan{TKey}"/> of <see cref="byte"/>s.
         /// </summary>
-        /// <param name="bytes">The <see cref="Span{TKey}"/> of <see cref="byte"/>s to create the PNG image from.</param>
+        /// <param name="bytes">The <see cref="ReadOnlySpan{TKey}"/> of <see cref="byte"/>s to create the PNG image from.</param>
         /// <returns>The QOI image as a PNG.</returns>
         /// <exception cref="Exception">If there is an invalid QOIF magic header or there was an error with stride width.</exception>
         public static SKBitmap GetImageFromSpan(ReadOnlySpan<byte> bytes) => GetImageFromSpan(bytes, out _);
 
-        /// <inheritdoc cref="GetImageFromSpan(System.ReadOnlySpan{byte})"/>
+        /// <inheritdoc cref="GetImageFromSpan(ReadOnlySpan{byte})"/>
+        /// <param name="bytes">The <see cref="ReadOnlySpan{TKey}"/> of <see cref="byte"/>s to create the PNG image from.</param>
```

Comment:
this causes the bytes description to exist twice.

Reply:
If I remove it, then Visual Studio shows this warning:
"CS1573 - Parameter 'bytes' has no matching param tag in the XML comment for 'QoiConverter.GetImageFromSpan(ReadOnlySpan<byte>, out int)' (but other parameters do)"

Reply:
yeah but it currently should stay like this. it warns because there's no explictit XML comment, but there's an implict one.
Either way, outside the scope of this.